davidpak/SmartNote
# Added PyPDF to environment.yml file

## What type of PR is this? (check all applicable)


## Description
Added PyPDF to the `environment.yml` file to avoid unnecessary imports.

## QA Instructions, Screenshots, Recordings
To create the environment, run:

`conda env create -f environment.yml`

To activate the environment run:

`conda activate smartnote_env`

To exit out of the environment run:

`conda deactivate smartnote_env`

To ensure that all dependencies are correctly installed run:

`pip list`

while in the conda environment.

## Added/updated tests?
_We encourage you to keep the code coverage percentage at 80% and above._

- [x] No, and this is why: _beep boop
